.github/workflows/tests-on-pr.yml

```diff
@@ -32,15 +32,11 @@ jobs:
           conda config --set always_yes yes
           --set changeps1 no
 
-      - name: Install libgtk for Linux
```

Comment:
Restore

Reply:
`libgtk` is no longer needed. here, wxpython is install via conda-forge as suggested.